content/architecture/infrastructure/deployment/altinn-apps/_index.md

```diff
@@ -18,92 +18,139 @@ The following diagram shows the deployment architecture for Altinn Studio togeth
-The deployment will be in the service owners own Kubernetes Cluster.
+### Deployment
+The Altinn App is deployed as sets of Docker Containers defined as Kubernetes Deployment.
+The deployment will be in the organisation's own Kubernetes Cluster.
 
-> All service owners will have their own Kubernetes Cluster.
+> All organisations will have their own Kubernetes Cluster.
 
-Each set of Kubernetes Deployment will be scaled based on the usage of the service.
+Each set of Kubernetes Deployment will be scaled based on the usage of the app.
 Scaling of the Deployment will be configurable in the Kubernetes Deployment and automatically scaled using
 [Kubernetes Replicaset](https://kubernetes.io/docs/reference/glossary/?all=true#term-replica-set).
 
-> Kubernetes Clusters and Serviceapps can do horizontal autoscaling.
+> Kubernetes Clusters and Altinn Apps can do horizontal autoscaling.
 > It is currently not decided if and how autoscaling will be used.
 
-The service container will consist of the runtime application and service specific code and configuration.
+The Altinn App in Kubernetes will be configured as a Kubernetes Service which consist of the runtime application
+and the Altinn App with related code and configuration.
 
-### Service Owner Clusters
-In Altinn Studio Apps, every service owner will have their separate Kubernetes Cluster in each environment.
+## Organisation Clusters for Altinn Apps
+
+In Altinn Apps, every organisation will have their separate Kubernetes Cluster in each environment.
 The Kubernetes Cluster architecture will be the same in test and production environment
-This will support scaling tailored each service owners needs and separation of services.
+This will support scaling tailored each organisations needs and separation of each organisation's app.
 
-#### Secrets
+### Secrets
 
-Pods, where Service Apps are delpoyed, gets access to secrets during deployment via HELM charts.
+Pods, where the Altinn App are delpoyed, gets access to secrets during deployment via HELM charts.
 
-#### Kubernetes Namespace
+### Kubernetes Namespace
 
 It's not planned to use namespaces.
 
-#### Routing
+### Kubernetes Service
+
+Each Altinn App will be a Kubernetes Service.
+
+### Routing
 
-To be able to route traffic to the correct container, each container is tagged to a specific
-end user service. The routing mecahnism routes to the correct container based on the url
-containing the service parameter.
+Routing in Kubernetes is handled by an Ingress-Controller.
 
-### Data services
+To be able to route traffic to the correct Kubernetes Service (Altinn App), each container is tagged to a specific
+Altinn App. The routing mecahnism routes to the correct Kubernetes Service based on the url
+containing the Altinn App parameter.
 
-The data services application is the application responsible exposing data related functionality 
-to the service containers. This container will be scaled based on need.
+## Data services
```

Comment:
Shouldnt this be a section under Platform cluster instead?

Reply:
Relocated.